agent/mapcall/src/main/java/randoop/instrument/CallReplacementTransformer.java

```diff
@@ -482,179 +455,221 @@ protected static void replace_instructions(
 
     // Remove the old handle. There should be no targeters left to it.
     try {
-      il.delete(ih);
+      instructionList.delete(instruction);
     } catch (Exception e) {
       throw new Error("Can't delete instruction", e);
     }
   }
 
   /**
-   * Returns true iff mgen is a constructor
+   * Indicates whether the method attribute is the local variable type table for the method.
    *
-   * @return true iff mgen is a constructor
+   * @param methodAttribute the method attribute
+   * @return true if the attribute is a local variable type table, false otherwise
    */
-  private boolean is_constructor(MethodGen mgen) {
-
-    if (mgen.getName().equals("<init>") || mgen.getName().equals("")) {
-      // log ("method '%s' is a constructor%n", mgen.getName());
-      return (true);
-    } else return (false);
-  }
-
-  public boolean is_local_variable_type_table(Attribute a) {
-    return (get_attribute_name(a).equals("LocalVariableTypeTable"));
-  }
-
-  /**
-   * Returns the attribute name for the specified attribute.
-   *
-   * @param a the attribute
-   * @return the name for the attribute
-   */
-  public String get_attribute_name(Attribute a) {
-
-    int con_index = a.getNameIndex();
+  private boolean isLocalVariableTypeTable(Attribute methodAttribute) {
+    int con_index = methodAttribute.getNameIndex();
     Constant c = pgen.getConstant(con_index);
-    String att_name = ((ConstantUtf8) c).getBytes();
-    return (att_name);
-  }
-
-  /** Class that reports tokenizing errors from the map file. */
-  static class MapFileErrorHandler extends StrTok.ErrorHandler {
-    LineNumberReader lr;
-    File map_file;
-
-    public MapFileErrorHandler(LineNumberReader lr, File map_file) {
-      this.lr = lr;
-      this.map_file = map_file;
-    }
-
-    @Override
-    public void tok_error(String s) {
-      throw new RuntimeException(
-          String.format("Error on line %d of %s: %s", lr.getLineNumber(), map_file, s));
-    }
+    return (((ConstantUtf8) c).getBytes().equals("LocalVariableTypeTable"));
   }
 
   /**
-   * Parse a method declaration. The declaration should be in the following format:
-   *
-   * <p>fully-qualified-method-name (args)
+   * Reads the file that specifies method calls that should be replaced by other method calls. The
+   * file is of the form:
```

Comment:
What is the relationship between this documentation and the manual?  They don't seem to be the same, and I think it's better to have documentation in one place to avoid inconsistencies.